### ISSUE TITLE
Ansible installation fails due to a missing setuptools

### ISSUE THREAD

**Issue by decentral1se** [OpenNebula/minione#78]
I had to run `pip install setuptools` to pass this firs step on a fresh Debian 10.

```
Install from PyPI 'ansible==2.9.9'  retry 1 retry 2 retry 3 FAILED
Collecting ansible==2.9.9
  Using cached ansible-2.9.9.tar.gz (14.2 MB)
--- STDERR ---
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-vIURiS/ansible/setup.py'"'"'; __file__='"'"'/tmp/pip-install-vIURiS/ansible/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-yV7JXF
         cwd: /tmp/pip-install-vIURiS/ansible/
    Complete output (3 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: No module named setuptools
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by xorel** [comment]
I can't reproduce, I have fixed #77. Can you try eventually once again with [master](https://github.com/OpenNebula/minione/blob/master/minione)?

**Comment by decentral1se** [comment]
I've since moved onto using other tools, thanks for your work anyway!
